api/src/main/java/marquez/MarquezConfig.java

```diff
@@ -38,4 +39,8 @@
   @Getter
   @JsonProperty("flyway")
   private final FlywayFactory flywayFactory = new FlywayFactory();
+
+  public static class GraphqlConfig {
+    @Getter private final boolean enabled = true;
```

Comment:
Should we disable graphql by default?

Reply:
I would add support to enable / disable graphql in [`marquez.yml`](https://github.com/MarquezProject/marquez/blob/main/marquez.example.yml). So, something like:

```yaml
graphql:
  enabled: ${GRAPHQL_ENABLED:-true}
```